docs/05-technical-reference/00-architecture/ac-01-application-connector-components.md

```diff
@@ -8,31 +8,11 @@ title: Application Connector components
 
 The Istio Ingress Gateway exposes Application Connector and other Kyma components.
```

Comment:
Regarding the architecture diagram: LGTM but it doesn't follow our [guidelines](https://kyma-project.io/community/guidelines/content-guidelines/09-diagrams/) 🤔

Reply:
I'm puzzled here 🤔 Why doesn't it? 
- Everything that means the same should look the same. ✅ 
- Limit visual noise.  ✅ 
- Keep it simple but descriptive.  ✅ 
- Follow the left-to-right direction when depicting the workflow.  ✅ 
- Always add an alternative (alt) text that concisely describes the content or function of the diagram you are referring to ✅ 
- Use [diagrams.net](https://www.diagrams.net/index.html) as a recommended tool. Export the diagram as an SVG and save it under the corresponding assets directory. ✅ 
- Keep your diagram reasonable in size ✅ 
- Keep the background of the diagram white  ✅ 
- Always add rounded secondary backgrounds to indicate the environment in which the workflow takes place. ✅ 
- Use mild blue (HEX: `#F0F6FF`) to indicate the main environment, such as a cluster, and mint green (HEX: `#DEF2DD`) to indicate subsidiary environments, such as Namespaces. ✅ 
- Use rounded rectangles as default box shapes. ✅ 
- Use white fill for main shapes, such as boxes ✅ 
- Use grey (HEX: `#666666`) for outlines. ✅ 
- Set the outlines of the shapes to 1pt. ✅ 
- Do not use any outlines for actors, steps, and secondary backgrounds. ✅ 
- Use black for both the primary and secondary texts.  ✅ 
- Use the following Helvetica font sizes:
   - 13pt for primary texts, such as shape names ✅  (_apart from one shape that was smaller and has font size of 12 pts, but I can fix this_)
- Always position the text horizontally. ✅ 
- When you add a title to a shape, put the text inside the shape. ✅ 
- Whenever possible, place the text in the central place of the shape and in the upper-central part of the background. ✅ 
- Use 1pt, rounded, grey (HEX: `#666666`) lines to connect shapes. ✅ 
- **Whenever you introduce an element that is different from other objects located in the diagram, include a reference key below the diagram to briefly explain the difference between the objects.** ✅ 

I can't see anything else...

Reply:
> Use mild blue (HEX: #F0F6FF) to indicate the main environment, such as a cluster, and mint green (HEX: #DEF2DD) to indicate subsidiary environments, such as Namespaces.

The color used in the diagram looks darker than mint green 🤔 Also, it's used to indicate a component (I assume Runtime Agent is on the same level as Application Gateway), not a subsidiary environment. As I said, it looks good so I won't insist on correcting that, it's just being inconsistent always worries me a bit 😅

Reply:
But this color is not supposed to indicate a subsidiary environment, but rather an exceptional component, and as a non-standard element, it's described in the Reference key.
Initially, I wanted to use a different color, but it didn't look good and I felt like it didn't draw enough attention, so I wanted to use a color that would match our color palette. 

But maybe you're right and the color is confusing...
Should I change it to purple? 
<img width="780" alt="image" src="https://user-images.githubusercontent.com/53601578/168305401-75ec7984-927b-4acb-8b4c-c3e196ced1ab.png">

Reply:
I did that. Please see whether it's better or if I should revert the change back to green.